librz/core/panels.c

```diff
@@ -4827,7 +4827,7 @@ void __panel_breakpoint(RzCore *core) {
 }
 
 void __panel_continue(RzCore *core) {
-	rz_core_cmd(core, "dc", 0);
+	rz_debug_continue(core->dbg);
```

Comment:
The command and this do slightly different things here :( I suggest you refactor the `dc` command in `cmd_debug.c` and wrap all the command does now within a `RZ_IPI rz_debug_continue_oldhandler (void *data, const char *input)` (it will be useful for when I/someone will convert existing debug commands to newshell).

Reply:
Thank you for the reply.
The overall idea is clear. But there are some particulars I'm not sure of.
So, I'll push a commit regarding this and let you request changes there.

> (it will be useful for when I/someone will convert existing debug commands to newshell)

Do tell if there are any others :)